src/js/load_fonts.js

```diff
@@ -0,0 +1,26 @@
+function loadFontByCreatingInvisibleDiv(fontFamily, dummyText) {
+  // Create a div element
+  const div = document.createElement("div");
+
+  // Set the font-family
+  div.style.fontFamily = fontFamily;
+
+  // Set the dummy text
+  div.textContent = dummyText;
+
+  // Make the div invisible
+  div.style.position = "absolute";
+  div.style.opacity = "0";
+  div.style.pointerEvents = "none";
+  div.style.zIndex = "-1";
+  div.style.left = "-1000px";
+  div.style.top = "-1000px";
+
+  // Append the div to the body
+  document.body.appendChild(div);
```

Comment:
Does building this placeholder in code pull it into the NPM package? The host webpage’s code may not be anticipating a placeholder div as a top-level child of the `<body>`, regardless of how it’s styled. Maybe it would be safer to insert this placeholder as a child of the map container?